cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
```

Comment:
Do we need to keep this sample file as part of the BCPL-assets source?

Reply:
I thought about that, probably not.